packages/extensions/venia-sample-backends/intercept.js

```diff
@@ -29,56 +27,61 @@ const fetchBackends = async debug => {
 };
 
 /**
- * Validation function to check if the backend being used is one of the sample backends provided
- * by PWA Studio. If yes, the function validates if the backend is active. If not, it reports an
- * error by calling the onFail function. In the error being reported, it sends the other sample
- * backends that the developers can use.
+ * A validation function that verifies that the backend used is active if it is
+ * one of the sample backends provided by Magento. If the backend is not active
+ * the function provides a list of alternatives.
  *
- * @param {Object} config.env - The ENV provided to the app, usually avaialable through process.ENV
+ * @param {Object} config.env - The ENV provided to the app, usually available through process.ENV
  * @param {Function} config.onFail - callback function to call on validation fail
  * @param {Function} config.debug - function to log debug messages in console in debug mode
  *
- * To watch the debug messages, run the command with DEBUG=*runEnvValidators*
+ * To watch the debug messages, run the command with DEBUG=pwa-buildpack:runEnvValidators
  */
 const validateSampleBackend = async config => {
-    console.warn(
-        "\n venia-sample-backends is a Dev only extension, please remove it from the project's package.json before going to production. \n"
-    );
-
     const { env, onFail, debug } = config;
 
-    const backendIsActive = await isBackendActive(env, debug);
-
-    if (!backendIsActive) {
-        debug(`${env.MAGENTO_BACKEND_URL} is inactive`);
+    const sampleBackends = await fetchBackends(debug);
```

Comment:
This swaps one network call on every yarn command for another, we don't want this either. I haven't pulled this down, but does this still run on every command still?

Reply:
You are right @tjwiebell. We should not be doing this. It will be running on every yarn command and that will be huge burden on the server.

Reply:
@tjwiebell oh jeez you are right. My intention was to only do an up-check against sample backends. But to obtain a list of sample backends to see if we're using one, I have to make a fetch.